cluster-up.sh

```diff
@@ -6,7 +6,7 @@ rm -fv kubeadm-join
 # Initialize the master.
 # The API server address is needed because the node has many network interfaces.
 # Save the console output, because it contains commands to execute afterwards.
-vagrant ssh k8s-master -c "sudo kubeadm init --apiserver-advertise-address 192.168.50.2 > /tmp/kubeadm-init"
+vagrant ssh k8s-master -c "sudo kubeadm init --pod-network-cidr=10.244.0.0/16 --apiserver-advertise-address 192.168.50.2 > /tmp/kubeadm-init"
```

Comment:
"The pod network CIDR is needed because ..."
"The pod network CIDR has this value because ..."

Reply:
This might be removed. I think i tried Flannel as i could not get weavenet to work, and flannel demands this pod-network-cidr to be specified. We can try and remove it, and see if it still works.

https://github.com/kubernetes/kubernetes/issues/36575